packages/wallet-service/tests/txProcessor.test.ts

```diff
@@ -515,300 +93,3 @@ describe('NFT metadata updating', () => {
-  ]);
-
-  await addToAddressBalanceTable(mysql, [
-    [addr, token, 2500, 0, null, 1, 0, 0, 2500],
-  ]);
-
-  await addToAddressTxHistoryTable(mysql, [
-    { address: addr, txId: txId1, tokenId: token, balance: 2500, timestamp: 0 },
-  ]);
-
-  await addToWalletBalanceTable(mysql, [{
-    walletId,
-    tokenId: token,
-    unlockedBalance: 2500,
-    lockedBalance: 0,
-    unlockedAuthorities: 0,
-    lockedAuthorities: 0,
-    timelockExpires: null,
-    transactions: 1,
-  }]);
-
-  const evt = JSON.parse(JSON.stringify(eventTemplate));
-  const tx = evt.Records[0].body;
-  tx.version = 1;
-  tx.tx_id = txId2;
-  tx.timestamp += timelock + 1;
-  tx.inputs = [createInput(2500, addr, txId1, 0, token)];
-  tx.outputs = [
-    createOutput(0, 2000, addr, token),    // one output to the same address
-    createOutput(1, 500, 'other', token),  // and one to another address
-  ];
-
-  // Adds txId2 that spends the utxo with index 0 from txId1
-  await txProcessor.onNewTxEvent(evt);
-
-  const evt2 = JSON.parse(JSON.stringify(eventTemplate));
-  const tx2 = evt2.Records[0].body;
-  tx2.version = 1;
-  tx2.tx_id = txId3;
-  tx2.timestamp += 1;
-  tx2.inputs = [createInput(2000, addr, txId2, 0, token)];
-  tx2.outputs = [
-    createOutput(0, 1500, addr, token),    // one output to the same address
-    createOutput(1, 500, 'other', token),  // and one to another address
-  ];
-
-  // Adds txId3 that spends the utxo with index 0 from txId2
-  await txProcessor.onNewTxEvent(evt2);
-
-  // Balance for addr should be 1500 and it should have 3 transactions (txId1, txId2 and txId3)
-  await expect(checkAddressBalanceTable(mysql, 2, addr, token, 1500, 0, null, 3)).resolves.toBe(true);
-
-  // Voids the first transaction (txId2), causing txId3 to be voided as well,
-  // as it spends utxos from txId2
-  await txProcessor.handleVoidedTx(tx);
-
-  // both utxos should be voided
-  await expect(checkUtxoTable(mysql, 5, txId2, 0, token, addr, 2000, 0, null, null, false, null, true)).resolves.toBe(true);
-  await expect(checkUtxoTable(mysql, 5, txId2, 1, token, 'other', 500, 0, null, null, false, null, true)).resolves.toBe(true);
-
-  // txId3 will be voided because txId2 was voided and it spends its utxo
-  await expect(checkUtxoTable(mysql, 5, txId3, 0, token, addr, 1500, 0, null, null, false, null, true)).resolves.toBe(true);
-
-  // the original utxo (txId1, 0) should not be voided and should not have been spent
-  await expect(checkUtxoTable(mysql, 5, txId1, 0, token, addr, 2500, 0, null, null, false, null, false)).resolves.toBe(true);
-
-  // Balance should be back to 2500 as the transactions that spent the original utxo were voided and we should
-  // have total of one transaction as both txId2 and txId3 were voided.
-  await expect(checkAddressBalanceTable(mysql, 2, addr, token, 2500, 0, null, 1)).resolves.toBe(true);
-}, 20000);
-
-test.skip('txProcessor should rollback the entire transaction if an error occurs on balance calculation', async () => {
```

Comment:
We are still rolling back when we fail to process a message, but now on different methods — `handleVoidedTx`, `handleVertexAccepted`, `handleUnvoidedTx`

We should re-create this test for all of them